entities/communication/quota.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "properties": {
+    "business_uid": {
+      "type": "string",
+      "description": "The business that the quota relates to."
```

Comment:
Unique identifier of the business that the quota relates to

Reply:
Unique identifier of the BusinessAccount that the quota relates to